<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1815</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Модель работы слесарей</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="line_AvgExcavatorWorkTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Мат. ожидание времени работы экскаваторов</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Затраты от простоя экскаватора</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ExcavatorLoss">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Прибыль от работы экскаватора</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ExcavatorProfit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Мат. ожидание времени работы бульдозера</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_AvgBuldozerWorkTime">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Затраты от простоя бульдозера</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_BuldozerProfit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_BuldozerLoss">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Прибыль от работы бульдозера</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>100</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Мат. ожидание времени, затрачиваемого на ремонт</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>0</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Зарплата слесаря 6 разр.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>0</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Зарплата слесаря 3 разр.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>190</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_BuldozerRepairMeanTime">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1.5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ExcavatorRepairMeanTime">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>250</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.25</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>170</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Слесарь 6 разр.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>230</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Вместе</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>130</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Экскаватор</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>130</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Бульдозер</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>1791</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Машина</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Начало работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Конец работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время простоя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Начало ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Конец ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Суммарное время работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Суммарное время ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Суммарное время простоя</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчет</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>160</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>1791</width>
      <height>241</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Машина</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Начало работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Конец работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время простоя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Начало ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Конец ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Суммарное время работы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Суммарное время ремонта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Суммарное время простоя</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1815</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
